morango/migrations/0010_auto_20170807_0123.py

```diff
@@ -0,0 +1,120 @@
+import django.utils.timezone
+from django.utils.timezone import utc
+import morango.utils.uuids
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('morango', '0009_auto_20170731_2142'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Nonce',
+            fields=[
+                ('id', morango.utils.uuids.UUIDField(editable=False, primary_key=True, serialize=False)),
+                ('timestamp', models.DateTimeField(default=django.utils.timezone.now)),
+                ('ip', models.CharField(blank=True, max_length=100)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.RemoveField(
+            model_name='syncsession',
+            name='connection_params',
+        ),
+        migrations.RemoveField(
+            model_name='transfersession',
+            name='chunksize',
+        ),
+        migrations.RemoveField(
+            model_name='transfersession',
+            name='records_remaining',
+        ),
+        migrations.AddField(
+            model_name='buffer',
+            name='source_id',
+            field=models.CharField(default=datetime.datetime(2017, 8, 7, 1, 22, 33, 701906, tzinfo=utc), max_length=96),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='certificate',
+            name='salt',
+            field=models.CharField(blank=True, max_length=32),
+        ),
+        migrations.AddField(
+            model_name='store',
+            name='source_id',
+            field=models.CharField(default='', max_length=96),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='syncsession',
+            name='active',
+            field=models.BooleanField(default=True),
+        ),
+        migrations.AddField(
+            model_name='syncsession',
+            name='is_server',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='syncsession',
+            name='local_instance',
+            field=models.TextField(default='{}'),
+        ),
+        migrations.AddField(
+            model_name='syncsession',
+            name='local_ip',
+            field=models.CharField(blank=True, max_length=100),
+        ),
+        migrations.AddField(
+            model_name='syncsession',
+            name='remote_instance',
+            field=models.TextField(default='{}'),
+        ),
+        migrations.AddField(
+            model_name='syncsession',
+            name='remote_ip',
+            field=models.CharField(blank=True, max_length=100),
+        ),
+        migrations.AddField(
+            model_name='transfersession',
+            name='last_activity_timestamp',
+            field=models.DateTimeField(blank=True, default=datetime.datetime(2017, 8, 7, 1, 23, 10, 795426, tzinfo=utc)),
```

Comment:
Since there are not yet any transfer sessions "in the wild", this will never actually get used. It asked me for a default for purposes of migration, and I gave it one.